searx/data/engine_descriptions.json

```diff
@@ -1229,6 +1308,14 @@
   "petalsearch news":[
    "petalsearch:en",
    "ref"
+  ],
+  "lib.rs":[
+   "Lib.rs is a catalog of programs and libraries written in the Rust programming language. It includes crates from the crates.io registry, and a few notable projects published only on GitHub or GitLab.",
+   "wikidata"
+  ],
+  "sourcehut":[
+   "software development repository and forge",
+   "wikidata"
   ]
```

Comment:
@joxcat FYI I added a commit on top to get descriptions of lib.rs and sourcehut / this by example is the English description.

> Normally you should already have the permissions to edit the branch, I checked the box while creating the PR

Oh yes, sorry it was my fail :-)